packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -76,14 +88,16 @@ export class StripeWebhookHandler extends StripeHandler {
       // Ensure the object is the latest version.
       const stripeObject = event.data.object as Record<string, any>;
       const resourceType =
-        STRIPE_OBJECT_TYPE_TO_RESOURCE[stripeObject.object as string];
+        ALLOWED_EXPAND_RESOURCE_TYPES[stripeObject.object as string];
```

Comment:
This would mean we are taking the `event.data.object` value as the latest version of this Stripe object, right? Isn't there a risk that the event object can be stale, and we would want to fetch the latest plan or product from Stripe and update the cache here? Something like `processWebhookEventToFirestore`, but for Redis.

Reply:
Only valid products and plans goes into the cache.  At this point we don't know if the product/plan is valid.

Reply:
But we also don't know if it's the latest version of that object, and if it is valid, we'll insert what could be a stale object into Redis downstream?